src/TropicalGeometry/variety_supertype.jl

```diff
@@ -15,9 +15,37 @@
 ###
 
 abstract type TropicalVarietySupertype{M,EMB} end
-function pm_object(v::TropicalVarietySupertype)
-  return v.polymakeTV
+# function pm_object(v::TropicalVarietySupertype)
+#   return v.polymakeTV
+# end
```

Comment:
Any reason to keep this around, commented out? Otherwise just remove it?

Reply:
No reason, I removed it.

Reply:
The way tropical varieties are built now it does not make sense to have `pm_object`, since the underlying object is not directly wrapping a polymake object, but a `PolyhedralComplex` instead. Occasionally these have a polymake bigobject pendant, but this is stored as an attribute.